src/utils/upload/processUpload.js

```diff
@@ -129,12 +131,25 @@ const processUpload = async (filesList, sampleType, samples, experimentId, dispa
   }, {});
 
   Object.entries(samplesMap).forEach(async ([name, sample]) => {
+    const { valid, verdict } = await inspectSample(sample);
+    const validationMessage = verdict.map((item) => `${verdictText[item]}`).join('\n');
+
     const filesToUploadForSample = Object.keys(sample.files);
 
+    if (!valid) {
+      pushNotificationMessage('error', `Error uploading sample ${name}: ${validationMessage}`);
+      return;
+    }
+
```

Comment:
I was thinking instead of showing a validation error we could show these errors in `FileUploadModal`. We already perform some checks in `onDrop`, so maybe just add that there?

That way they can see more easily exactly what file is wrong and how instead of getting a lot of push notifications.

Reply:
We can't, because the files aren't yet uploaded in the file upload modal, and we need to open the file to check its validity

Reply:
After speaking in a call, we can do this (we don't need the files to be uploaded to perform these checks) but it involves some ux design steps first so it was decided to move it to another ticket so I am resolving this comment